SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -179,6 +179,7 @@ void UGlobalStateManager::ApplyCanBeginPlayUpdate(bool bCanBeginPlayUpdate)
 	// For now, this will only be called on non-authoritative workers.
 	if (bCanBeginPlay)
 	{
+		NetDriver->World->GetWorldSettings()->SetGSMReadyForPlay();
```

Comment:
Can this line be moved into TriggerBeginPlay. I would rather it not be set twice.